clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -2951,7 +2951,6 @@ protected StoreVersionState waitVersionStateAvailable(String kafkaTopic) throws
       elapsedTime = System.currentTimeMillis() - startTime;
 
       if (state != null) {
-        LOGGER.info("Version state is available for {} after {} ms", kafkaTopic, elapsedTime);
```

Comment:
Looks like it will be called only in chunking store. Do you think it is worth using redundant filter or it is completely not worth logging here?

Reply:
Actually I think we can reduce the logging in another way, we call the getVersionState() first one time, if it is null then we enter the for loop and thus the logging will only be once per SIT...but maybe it is fine just to remove it completely.

Reply:
I vote to remove it completely.

Reply:
Yeah, if having the log doesn't help, I also would like to remove it. If there are issues, it will be captured by the error log eventually